sources/packages/backend/workflow/src/workflow-definitions/assessment-gateway.bpmn

```diff
@@ -306,7 +306,7 @@
     <bpmn:serviceTask id="save-assessment-data-part-time-task" name="Save assessment output">
       <bpmn:extensionElements>
         <zeebe:ioMapping>
-          <zeebe:input source="={&#10;  weeks: offeringWeeks,&#10;  tuitionCost: offeringActualTuitionCosts,&#10;  booksAndSuppliesCost: offeringProgramRelatedCosts,&#10;  mandatoryFees: calculatedDataMandatoryFees,&#10;  exceptionalEducationCost: offeringExceptionalExpenses,&#10;  childcareCost: calculatedDataTotalChildCareCost,&#10;  totalFamilyIncome: calculatedDataTotalFamilyIncome,&#10;  transportationCost: calculatedDataTotalTransportationAllowance,&#10;  miscellaneousAllowance: calculatedDataMiscellaneousAllowance,&#10;  totalAssessmentNeed: calculatedDataTotalAssessedNeed,&#10;  totalAcademicExpenses: calculatedDataTotalAcademicExpenses,&#10;  finalFederalAwardNetCSGPAmount: finalFederalAwardNetCSGPAmount,&#10;  finalFederalAwardNetCSPTAmount: finalFederalAwardNetCSPTAmount,&#10;  finalFederalAwardNetCSGDAmount: finalFederalAwardNetCSGDAmount,&#10;  finalProvincialAwardNetBCAGAmount: finalProvincialAwardNetBCAGAmount,&#10;  finalProvincialAwardNetSBSDAmount: finalProvincialAwardNetSBSDAmount,&#10;  finalFederalAwardNetCSLPAmount: finalFederalAwardNetCSLPAmount,&#10;  finalAwardTotal: finalAwardTotal&#10;}" target="assessmentData" />
+          <zeebe:input source="={&#10;  weeks: offeringWeeks,&#10;  tuitionCost: calculatedDataActualTuitionCosts,&#10;  booksAndSuppliesCost: calculatedDataProgramRelatedCosts,&#10;  mandatoryFees: calculatedDataMandatoryFees,&#10;  exceptionalEducationCost: offeringExceptionalExpenses,&#10;  childcareCost: calculatedDataTotalChildCareCost,&#10;  totalFamilyIncome: calculatedDataTotalFamilyIncome,&#10;  transportationCost: calculatedDataTotalTransportationAllowance,&#10;  miscellaneousAllowance: calculatedDataMiscellaneousAllowance,&#10;  totalAssessmentNeed: calculatedDataTotalAssessedNeed,&#10;  totalAcademicExpenses: calculatedDataTotalAcademicExpenses,&#10;  finalFederalAwardNetCSGPAmount: finalFederalAwardNetCSGPAmount,&#10;  finalFederalAwardNetCSPTAmount: finalFederalAwardNetCSPTAmount,&#10;  finalFederalAwardNetCSGDAmount: finalFederalAwardNetCSGDAmount,&#10;  finalProvincialAwardNetBCAGAmount: finalProvincialAwardNetBCAGAmount,&#10;  finalProvincialAwardNetSBSDAmount: finalProvincialAwardNetSBSDAmount,&#10;  finalFederalAwardNetCSLPAmount: finalFederalAwardNetCSLPAmount,&#10;  finalAwardTotal: finalAwardTotal&#10;}" target="assessmentData" />
```

Comment:
Do we require the exceptional expense for the PT assessment data at all? 
![image](https://github.com/user-attachments/assets/ad657506-c5d2-496e-862f-cf3523403db9)

Reply:
We do not require it in the NOA but we do for some data inventory reports (that may be removed in the future).
I would not remove it right now.